src/params.h

```diff
@@ -80,7 +80,16 @@ code path */
 ********************************************************************************
 *******************************************************************************/
 
-#define MFAKTC_VERSION "0.23.3" /* Do not change without consensus! */
+/*
+ MFAKTC_VERSION sets the version number. You must make sure the version string
+ complies with the semantic versioning scheme: https://semver.org
+
+ Otherwise, the automated builds could fail in GitHub Actions.
+
+ Please discuss with the community before making changes to version numbers!
+ */
+
+#define MFAKTC_VERSION "0.23.3"
```

Comment:
Everything looks good to me. I've testes CUDA 12.8.1/Windows build a bit and everything seems to be good to go. I think this PR worth a new version:

```suggestion
#define MFAKTC_VERSION "0.23.4"
```